js/example-sim-main.ts

```diff
@@ -0,0 +1,47 @@
+// Copyright 2013-2024, University of Colorado Boulder
+
+/**
+ * Main entry point for the sim.
+ *
+ * @author Chris Malley (PixelZoom, Inc.)
+ * @author Sam Reid (PhET Interactive Simulations)
+ */
+
+import Sim, { SimOptions } from '../../joist/js/Sim.js';
+import simLauncher from '../../joist/js/simLauncher.js';
+import Tandem from '../../tandem/js/Tandem.js';
+import ExampleSimStrings from './ExampleSimStrings.js';
+import './common/ExampleSimQueryParameters.js';
+import MagnetsScreen from './magnets/MagnetsScreen.js';
+import ParticlesScreen from './particles/ParticlesScreen.js';
+
+// Launch the sim. Beware that scenery Image nodes created outside simLauncher.launch() will have zero bounds
+// until the images are fully loaded. See https://github.com/phetsims/coulombs-law/issues/70#issuecomment-429037461
+simLauncher.launch( () => {
+
+  const titleStringProperty = ExampleSimStrings[ 'example-sim' ].titleStringProperty;
+
+  const screens = [
+    new MagnetsScreen( { tandem: Tandem.ROOT.createTandem( 'magnetsScreen' ) } ),
```

Comment:
I included the tandem stuff because create-sim adds it, and I wanted example-sim to match that. I guess we want example-sim to diverge from simula-rasa in this respect?